suite/tests/linux/clone.c

```diff
@@ -48,6 +48,7 @@
 #include <errno.h>
```

Comment:
Q: Is the test failing by not building successfully, or by running incorrectly? I assume it's the latter because the issue description says "The linux.clone* tests should not fail if clone3 returns ENOSYS." But I would have expected a missing struct definition to cause a build failure.

I'm probably missing some context. Could you clarify in the PR description?

Reply:
The test is failing due to build errors.